### ISSUE TITLE
isSymbolMarketOpen() This operation is not supported

### ISSUE THREAD

**Issue by dbvcode** [Reiryoku-Technologies/Mida-cTrader#19]
`MidaUnsupportedOperationError: This operation is not supported`
I can call it like:
```
await account.isSymbolMarketOpen(symbol)
```
or 
```
sym = await account.getSymbol(symbol)
await sym.isMarketOpen()
```

it does not make a difference, the error is the same.
I assume the method is not implemented yet for cTrader.

**Comment by Vasile-Peste** [comment]
It has been implemented with `cTrader 4.0.0` https://github.com/Reiryoku-Technologies/Mida-cTrader/blob/v4.0.0/CHANGELOG.md

**Comment by dbvcode** [comment]
Awesome!

**Comment by dbvcode** [comment]
It does not seem to be returning what it should.
I have GER40 on Pepperstone that's open and `isSymbolMarketOpen('GER40')` returns `false`.

**Comment by Vasile-Peste** [comment]
You are right, there was a bug, only the day number was considered and not the hours, minutes, seconds...
This should be fixed now with [Mida 2022.1](https://github.com/Reiryoku-Technologies/Mida/releases/tag/v2022.1.0) and [Mida cTrader 5](https://github.com/Reiryoku-Technologies/Mida-cTrader/releases/tag/v5.0.0), let me know if it works!

**Comment by dbvcode** [comment]
Seems to work! Thanks
I'll see over the weekend what happens!
